webprojects/SUMO.md

```diff
@@ -0,0 +1,77 @@
+* Translated strings from sumo-l10n are pushed to production.
+
+There are three ways to test the product:
+
+* Staging server: https://support.allizom.org​.
+* Production server: http://support.mozilla.org/.
+* Locally: if you want to test your update in a fast and independent way, you should [install your own local instance of Kitsune](https://kitsune.readthedocs.io/en/latest/localization.html#testing-localized-strings) on your computer.
+
+### Testing on production
+
+Localization updates are pushed on Thursdays on staging for review in context, and on Mondays on production. Either way, the SUMO team relies on you to find issues and report bugs that are usually fixed at a timely manner.
+
+### Testing locally
+
+According to SUMO developers, Kitsune is in a very fragile state. The SUMO team has very limited resources to support local installs. Unless you are technical and self sufficient, you are strongly advised to use Pontoon and staging server to preview your localization work.
+
+If you choose to give local install a try, follow the step by step instruction on [testing localized SUMO locally]((https://kitsune.readthedocs.io/en/latest/localization.html#testing-localized-strings), including local installation of Kitsune, file compilation and steps to commit changes to repository.
+
+## FAQ
+
+For any additional questions not covered here, you can contact [Michał](mailto:mdziewonski@mozilla.com).
+
+### How often are new strings added? Is there a sprint?
+
+No, there are no sprints. Strings are added at **as needed** basis. Whenever a release or change is required, new strings will be added to both Pontoon and Kitsune on Mondays.
+
+### How do I participate in the [SUMO l10n forum](https://support.mozilla.org/en-US/forums/l10n-forum)?
+
+To be part of the SUMO community, you need to [create a SUMO account](https://support.mozilla.org/en-US/users/authcontributor), which allows you to write in our community l10n forum and localize any articles you see in SUMO. This is separate from the account used to log into Pontoon in order to localize SUMO product and other Mozilla products and projects.
+
+### Are there email communications when new strings are added?
+
+Only when there is a release or a major change will there be communications to localizers through [mailing list](sumo-locale-leaders@), direct private message, and the SUMO l10n forum. The Pontoon dashboard is the best way to check for pending task.
+
+### What are the ways to report an issue?
+
+The best place to report issues is by filing [bugs to SUMO](https://bugzilla.mozilla.org/enter_bug.cgi?product=support.mozilla.org&component=Localization). This is the quickest way to have a problem resolved.
+
+Alternatively, you can report a problem through the SUMO l10n forum. You can also raise questions through [web projects mailing list](dev-l10n-web@lists.mozilla.org), [IRC](#sumo), and other social channels. In the end, issues identified will be tracked through Bugzilla.
```

Comment:
drop the link on IRC, or point it to https://wiki.mozilla.org/IRC, e.g. `in the #sumo channel on [IRC](https://wiki.mozilla.org/IRC)`